utils/symbolicate-linux-fatal.py

```diff
@@ -0,0 +1,119 @@
+#!/usr/bin/env python
```

Comment:
Please rename the tool omitting the `.py` suffix.  Users have no necessity to know which language the tool is written in.